game/game.wren

```diff
@@ -105,15 +105,133 @@ class Main {
             }
         }
 
+
+        if(engine.GetInput().DebugGetKey(Keycode.eN())){
+           cheats.noClip = !cheats.noClip
+        }
+        // MOVEMENT
+        if(cheats.noClip == true){
+            return
```

Comment:
we're also still in the tick function, returning early will introduce bug quickly. You should split up movement to its own wren file.